numpy/core/src/common/lowlevel_strided_loops.h

```diff
@@ -191,60 +202,15 @@ PyArray_GetDTypeTransferFunction(int aligned,
                             npy_intp src_stride, npy_intp dst_stride,
                             PyArray_Descr *src_dtype, PyArray_Descr *dst_dtype,
                             int move_references,
-                            PyArray_StridedUnaryOp **out_stransfer,
-                            NpyAuxData **out_transferdata,
+                            NPY_cast_info *cast_info,
                             int *out_needs_api);
```

Comment:
You can remove `out_stransfer` and `out_transferdata` from the list of paramters in the comment above